source/MenuPanel.cpp

```diff
@@ -73,7 +72,9 @@ MenuPanel::MenuPanel(PlayerInfo &player, UI &gamePanels)
 
 void MenuPanel::Step()
 {
-	if(GetUI()->IsTop(this) && alpha < 1.f)
+	if(GetUI()->IsTop(this))
+		creditsScrolling = true;
```

Comment:
Do we really need this boolean? What does it add that the original condition didn't, except needing to explicitly set it to false when we push a panel on top?

Reply:
Well the original condition doesn't really work anymore because of the new panel.

So it's either this (which is the same behavior as master) or:

- credits starts scrolling after the circle is gone, and credits pause when opening day the preference panel, or
- credits starts scrolling immediately but opening panels doesn't pause the credits

Reply:
NVM, I found an alternative.